ui/packages/image/src/StaticImage.svelte

```diff
@@ -19,6 +20,15 @@
 {#if value === null}
 	<Empty size="large" unpadded_box={true}><Image /></Empty>
 {:else}
+	<div class="download">
+		<a
+			href={value}
+			target={window.__is_colab__ ? "_blank" : null}
+			download={"image"}
```

Comment:
We don't have the filename generated on the python side so I'm using `image` as the filename. I'm not sure we'd even want the tempfile name here anyways.

Reply:
Can we use the stem from `{value}` here as the filename? I'll be working on #943 probably next week which will make temporary file names be the same as the original filenames (in those cases where we have the original filenames) so I think it's better if we use the temp names.

Reply:
Hold on, take that back. `{value}` is base64 isn't it? I think we don't create temporary files for images like we do for the other components

Reply:
So nvm, this PR LGTM